python/ray/autoscaler/_private/util.py

```diff
@@ -604,6 +604,9 @@ def parse_usage(usage: Usage, verbose: bool) -> List[str]:
         if "node:" in resource:
             continue  # Skip the auto-added per-node "node:<ip>" resource.
```

Comment:
This makes me think we should use a name like `node:head` to be more consistent with other resources?

Reply:
Good point.